src/api/verification_library.py

```diff
@@ -100,3 +100,37 @@ def get_library_item(self, item_name: str) -> Dict:
         }
 
         return item_dict
+
+    def get_required_datapoints_by_library_items(
+        self, datapoints: List[str] = []
+    ) -> Union[Dict, None]:
+        """Summarize datapoints that need to be used to support specified library items. Use this function with caution as it 1) requires aligned data points naming across all library items; 2) does not check the topological relationships between datapoints.
+
+        Args:
+            items: list of str, default []. Library items to summarize datapoints from. By default, summarize all library items loaded at instantiation.
+
+        Returns:
+            Dict with keys being the datapoint name and values being a sub Dict with the following keys:
+                - number_of_items_using_this_datapoint: int, number of library items that use this datapoint.
+                - library_items_list: List, of library item names that use this datapoint.
+        """
+
+        if not isinstance(datapoints, List):
+            logging.error(
+                f"The `datapoints` argument type must be List, but {type(datapoints)} type is provided."
+            )
+            return None
```

Comment:
Should we add validation input to make sure that `datapoints` is not an empty list?

Reply:
good point! Addressed.